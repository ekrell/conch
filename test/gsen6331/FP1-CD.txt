5.790000000000000000e+02,1.870000000000000000e+02
5.460000000000000000e+02,2.180000000000000000e+02
4.690000000000000000e+02,4.000000000000000000e+02
5.230000000000000000e+02,4.700000000000000000e+02
3.180000000000000000e+02,7.440000000000000000e+02
4.300000000000000000e+02,9.220000000000000000e+02
4.360000000000000000e+02,9.320000000000000000e+02
4.670000000000000000e+02,9.560000000000000000e+02
4.850000000000000000e+02,9.890000000000000000e+02
