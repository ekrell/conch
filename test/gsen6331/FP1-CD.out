Using input region raster: test/gsen6331/full.tif
      input graph file: test/gsen6331/full_evg-a_fp1.pickle
  Start: (42.32343, -70.99428)
    End: (42.336, -70.88737)
Incorporating forces into energy cost:
  u: test/gsen6331/waterMag.tif
  v: test/gsen6331/waterDir.tif
Done solving with a*, 124.00064039230347 seconds
Planning results (with currents):
    Distance: 11.9613 km
    Duration: 39.8710 min
    Cost: 262.4711
